Formula/m/mysql-client@8.4.rb

```diff
@@ -0,0 +1,61 @@
+class MysqlClientAT84 < Formula
+  desc "Open source relational database management system"
+  homepage "https://dev.mysql.com/doc/refman/8.4/en/"
+  url "https://cdn.mysql.com//Downloads/MySQL-8.4/mysql-8.4.2.tar.gz"
```

Comment:
Can be done as a follow-up:

```suggestion
  url "https://cdn.mysql.com/Downloads/MySQL-8.4/mysql-8.4.2.tar.gz"
```